azurerm/resource_arm_api_management_group.go

```diff
@@ -0,0 +1,165 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/apimanagement/mgmt/2018-06-01-preview/apimanagement"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmAPIManagementGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmAPIManagementGroupCreateUpdate,
+		Read:   resourceArmAPIManagementGroupRead,
+		Update: resourceArmAPIManagementGroupCreateUpdate,
+		Delete: resourceArmAPIManagementGroupDelete,
```

Comment:
_minor_ the other resources use 
```suggestion
		Delete: resourceArmApiManagementGroupDelete,
```